addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -854,8 +854,10 @@
                             <field name="company_id" groups="base.group_multi_company" readonly="state != 'draft'"/>
                             <field name="user_id" widget="many2one_avatar_user" readonly="state != 'draft'"/>
                             <field name="accounting_date"
+                                   string="Expense Report Date"
+                                   help="Specify the bill date of the related vendor bill."
```

Comment:
Better to define it on the field itself IMO
Else other views would not have that change